doc/python/axes.md

```diff
@@ -41,7 +41,7 @@ Other kinds of subplots and axes are described in other tutorials:
 - [Polar axes](/python/polar-chart/). The axis object is [`go.layout.Polar`](/python/reference/layout/polar/)
 - [Ternary axes](/python/ternary-plots). The axis object is [`go.layout.Ternary`](/python/reference/layout/ternary/)
 - [Geo axes](/python/map-configuration/). The axis object is [`go.layout.Geo`](/python/reference/layout/geo/)
-- [Mapbox axes](/python/mapbox-layers/). The axis object is [`go.layout.Mapbox`](/python/reference/layout/mapbox/)
+- [Map axes](/python/map-layers/). The axis object is [`go.layout.Map`](/python/reference/layout/map/)
```

Comment:
@LiamConnors 
Is `/python/map-layers/` correct? Or we should use `mapbox`?

Reply:
good catch. was a typo